components/pluralrules/benches/fixtures/mod.rs

```diff
@@ -0,0 +1,47 @@
+use icu_locale::LanguageIdentifier;
+use icu_pluralrules::PluralCategory;
+use std::collections::HashMap;
+
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct NumbersFixture {
```

Comment:
Here and elsewhere under `benches/`: Why are these `pub`?  Maybe `pub(crate)` is better?

Reply:
benches and tests modules are not available anywhere else, do I never paid attention to their visibility. They're not accessible from anywhere else.

Also, even in `src/` code if a module has public methods but the parent module doesn't expose that module publicly, then it is not part of the public API.

I prefer to control API visibility per module, then per function/struct/enum etc.

Reply:
I'm going to mark each function and struct as `pub(crate)` but I think it is superfluous.